test/mongoose_push_test.exs

```diff
@@ -147,12 +147,14 @@ defmodule MongoosePushTest do
         tag: string(min: 3, max: 15, chars: :ascii),
         sound: string(min: 3, max: 15, chars: :ascii),
         click_action: string(min: 3, max: 15, chars: :ascii),
+        time_to_live: int(min: 0, max: 4500),
         priority: choose(from: [value(:normal), value(:high)]),
       ], repeat_for: 10 do
 
       notification =
         %{:service => :fcm,
           :priority => priority,
+          :time_to_live => time_to_live,
```

Comment:
There is no actual check whether this is correctly set in the notification. We need that tested.